### DISCUSSION TITLE
PrimeVue Full Calendar like release

### DISCUSSION THREAD

**Discussion by m-drony** [primefaces/community#418] [PrimeVue]
Hi there,

According to https://github.com/primefaces/primevue/issues/3827#issuecomment-1504884747 there was an intent to release a Full Calendar like PrimeVue component.

Can you tell us if it's still in your roadmap and if so do you have a release date/version ?

Thanks,

Matt

**Comment by tugcekucukoglu** [comment]
Hi,

It's definitely on our roadmap.

Thanks,

**Reply by m-drony** [reply]
Hi,

Thanks for your answer, no chance to have a hint about a release date ? :) (quarter/semester ?)

Matt

**Comment by BITSOLVER** [comment]
The Roadmap still shows Q2 2025, is there any update on this?

Steve

**Comment by fonziemeagle** [comment]
Any idea when the Full Calendar feature will be completed?  Looking at the roadmap for 2026, it is showing for Q1.  I see that it keeps sliding...

**Comment by cagataycivici** [comment]
It is implemented and being tested at the moment. Note that it is part of PrimeUI PRO commercial suite components not PrimeVue. It will provide a Vue version in addition to Angular, React and more alternatives like Web Components, Blazor in the future.

**Reply by bbevren** [reply]
Will it be avaliable in primestore?

**Reply by cagataycivici** [reply]
Yes, Scheduler (FullCalendar) launches with the first phase that also includes TextEditor, Charts and PDF Viewer.

 Phase 2 will bring DataGrid, Diagram, FileManager, Gantt, Task Board (Kanban).

Each component can be purchased standalone or the whole PrimeUI PRO Suite. The Vue components are standalone and do not depend on PrimeVue APIs by an architectural decision so for example, Scheduler can work in any Vue application without adding dependencies e.g. PrimeVue.
